docs/roman/pipeline/exposure_pipeline.rst

```diff
@@ -73,12 +74,15 @@ for errors and Data Quality flags created and initialized to zero.
 Outputs
 ----------
 
-2D corrected ramp
-+++++++++++++++++
+2D Image model
+++++++++++++++
 
-:Data model: `~romancal.datamodels.RampModel`
-:File suffix: _ramp
+:Data model: `~romancal.datamodels.ImageModel`
+:File suffix: _cal
 
-Result of applying all pipeline steps up through the :ref:`jump <jump_step>` step,
-to produce corrected and CR-flagged 2D ramp data, which will have one less data dimensions
-as the input raw 3D data (ncols x nrows x ngroups ).
+Result of applying all pipeline steps up through the
+:ref:`flatfield <flatfield_step>` step, to produce corrected flatfield data
+which is 2D image data, which will have one less data dimensions as the input
+raw 3D data (ncols x nrows x ngroups). In addition to being a 2-dimensional
```

Comment:
The order of dimensions should be: ngroups x ncols x nrows

Reply:
done